scripts/spack/packages/serac/package.py

```diff
@@ -60,6 +60,7 @@ class Serac(CachedCMakePackage, CudaPackage, ROCmPackage):
     variant("profiling", default=False, 
             description="Build with hooks for Adiak/Caliper performance analysis")
 
+    variant("enzyme", default=True, description="Enable Enzyme Automatic Differentiation Framework") 
```

Comment:
should we have a conflict for if the compiler isn't clang but +enzyme?

or more strictly, always conflict if not clang?

Reply:
Yes we should have conflicts for all non-llvm-based compilers when this merged.. I'll add it.  There should also be a conflict put into the Enzyme package. I have spoken to the Spack team on how to express that.